### PR TITLE
No reminder emails

### PR DESCRIPTION
**Pull request by calellowitz**
@nickpell cc: @gcapalbo 
http://manage.dimagi.com/default.asp?226815#1151744

### CODE REVIEW

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1016,7 +1016,8 @@ class Subscription(models.Model):
     is_active = models.BooleanField(default=False)
     do_not_invoice = models.BooleanField(default=False)
     no_invoice_reason = models.CharField(blank=True, null=True, max_length=256)
-    do_not_email = models.BooleanField(default=False)
+    do_not_email_invoice = models.BooleanField(default=False)
+    do_not_email_reminder = models.BooleanField(default=False)
```
Comment (gcapalbo) [inline corehq/apps/accounting/models.py]: i don't think it's worth it to change now, but it's generally better to name boolean properties in the positive (i.e., `send_email_invoice` and `send_email_reminder`)

**Comment by gcapalbo** [comment]
build failure looks related to this change, but otherwise :+1: on :white_check_mark: from me. will let @nickpell sign off

**Comment by nickpell** [comment]
definitely related

```
======================================================================
ERROR: test suite for <corehq.tests.nose.HqdbContext object at 0x7f6e39d24650>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 296, in setUp
    self.setupContext(context)
  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/mnt/corehq/tests/nose.py", line 346, in setup
    super(HqdbContext, self).setup()
  File "/vendor/lib/python2.7/site-packages/django_nose/plugin.py", line 428, in setup
    self.old_names = self.runner.setup_databases()
  File "/vendor/lib/python2.7/site-packages/django/test/runner.py", line 109, in setup_databases
    return setup_databases(self.verbosity, self.interactive, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/test/runner.py", line 299, in setup_databases
    serialize=connection.settings_dict.get("TEST", {}).get("SERIALIZE", True),
  File "/vendor/lib/python2.7/site-packages/django/db/backends/creation.py", line 377, in create_test_db
    test_flush=True,
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 115, in call_command
    return klass.execute(*args, **defaults)
  File "/vendor/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/vendor/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 161, in handle
    executor.migrate(targets, plan, fake=options.get("fake", False))
  File "/vendor/lib/python2.7/site-packages/django/db/migrations/executor.py", line 68, in migrate
    self.apply_migration(migration, fake=fake)
  File "/vendor/lib/python2.7/site-packages/django/db/migrations/executor.py", line 102, in apply_migration
    migration.apply(project_state, schema_editor)
  File "/vendor/lib/python2.7/site-packages/django/db/migrations/migration.py", line 108, in apply
    operation.database_forwards(self.app_label, schema_editor, project_state, new_state)
  File "/vendor/lib/python2.7/site-packages/django/db/migrations/operations/special.py", line 117, in database_forwards
    self.code(from_state.render(), schema_editor)
  File "/mnt/corehq/apps/accounting/migrations/0012_billing_metadata_data_migration.py", line 9, in migrate_metadata
    for subscription in Subscriptions.objects.all():
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 141, in __iter__
    self._fetch_all()
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 966, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 265, in iterator
    for row in compiler.results_iter():
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 701, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 787, in execute_sql
    cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/vendor/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
ProgrammingError: column accounting_subscription.do_not_email does not exist
LINE 1: ...son", "accounting_subscription"."funding_source", "accountin...
                                                             ^
```

**Comment by calellowitz** [comment]
:white_check_mark: @nickpell

**Comment by nickpell** [comment]
good work on fixing the test error!

**Pull request merged**